access/src/main/java/edu/unc/lib/dl/ui/controller/SearchActionController.java

```diff
@@ -117,6 +133,22 @@ private String formatQueryText(HttpServletRequest request) {
         searchRequest.setRootPid(pid);
```

Comment:
The body of this method is basically the same as the other searchJson one, would it be possible to make one call the other?